businesscentral-runtime/scripts/businesscentral-runtime/install

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+SOURCES_DIR=/tmp/artifacts
+
+unzip ${SOURCES_DIR}/jboss-bpmsuite-7.0.0.redhat-20170824.071756-1-business-central-runtime-eap7.zip -d ${SOURCES_DIR}
```

Comment:
This artifact name is an initial approach. It will probably change.

Reply:
jboss-bpmsuite-7.0.0-business-central-monitoring-eap7.zip